shell/common/rasterizer.cc

```diff
@@ -192,9 +192,9 @@ void Rasterizer::Draw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline,
 
   // Merging the thread as we know the next `Draw` should be run on the platform
   // thread.
-  if (surface_ != nullptr && surface_->GetExternalViewEmbedder() != nullptr) {
-    surface_->GetExternalViewEmbedder()->EndFrame(should_resubmit_frame,
-                                                  raster_thread_merger_);
+  if (surface_ && external_view_embedder_) {
```

Comment:
Do we still want to predicate this on `surface_`?  Looks like it should work without one.

Reply:
Done!